frontend/src/assets/css/variables.scss

```diff
@@ -15,20 +15,18 @@
 // (Using a different license header format here because scss-to-json doesn't
 // like multi-line comments)
 
-// 1rem == 16px
-// (Can't set actual pixels here because it breaks reboot)
-$font-size-base: 1rem;
+$font-size-base: 18px;
 
 // Overwrite other theme settings.
 $border-radius: 0.25rem;
 $border-radius-sm: 0.25rem;
 
 // These variables are needed for reboot. We supply them from bootstrap's
 // variable files
-$small-font-size: 0.875em !default;
-$sub-sup-font-size: 0.75em !default;
+$small-font-size: 14px !default; // Same as fontSizeSmall in typography.ts
+$sub-sup-font-size: 12px !default; // Same as fontSizeTwoSmall in typography.ts
```

Comment:
Same here. This is originally from Bootstrap's variable files, but if the base is set in the HTML tag, shouldn't be easily an `rem`

Reply:
> but if the base is set in the HTML tag, shouldn't be easily an rem

These are used in `reboot.scss` to set the size of `<small>` and `<sub>`/`<sub>`. If I change them to `rem` they'll have to be something like `0.777778rem` and `0.666667rem`. Is that OK with you?

By the way, I'd love for us to move these variables from SASS to CSS `--custom-properties`. This way we could share them across CSS and TS files directly.

Reply:
Yea. That could work. There's a couple Sass functions that we use. We can try to remove them or recreate them not as a function.

Reply:
Want me to do this in this PR? Or is that a future improvement?